nni/tools/nnictl/launcher.py

```diff
@@ -608,8 +608,7 @@ def create_experiment(args):
     try:
         launch_experiment(args, experiment_config, 'new', experiment_id)
     except Exception as exception:
-        nni_config = Config(experiment_id)
-        restServerPid = nni_config.get_config('restServerPid')
+        restServerPid = Experiments().get_all_experiments().get(experiment_id).get('pid')
```

Comment:
If get(experiment_id) returns `None`, nnictl will crash here.

Reply:
indeed, fix it